iphone/Maps/Classes/CustomAlert/Toast/Toast.swift

```diff
@@ -7,11 +7,23 @@ final class Toast: NSObject {
   }
 
   private var blurView = UIVisualEffectView(effect: UIBlurEffect(style: .dark))
+  private var timer: Timer?
+
+  private static var toasts: [Toast] = []
 
   @objc static func toast(withText text: String) -> Toast {
-    return Toast(text)
+    let toast = Toast(text)
+    toasts.append(toast)
+    return toast
   }
-
+  
+  @objc static func hideAll(){
+    var toastsCopy = toasts
```

Comment:
Why a copy is needed?

Reply:
Was afraid of mutating the array while iterating over it. Doing a little looking it seems like it doesn't matter so i can make the change